packages/@atjson/hir/package.json

```diff
@@ -18,6 +18,6 @@
     "@atjson/document": "file:../document"
   },
   "peerDependencies": {
-    "@atjson/document": "^0.23.0"
+    "@atjson/document": ">=0.23.0"
```

Comment:
I'm confused here - the old peer dependency version was any version from 0.23.0 and less than 1.0.0. We never released a 1.0.0 version so what was breaking before that this is fixing?

Reply:
Oh  NVM, I didn't know  what `^` means exactly. TIL it means any new version  that doesn't update the left-most non-zero version, not  necessarily the major version, so `^0.23.0` are versions `0.23.x` and not `0.24.0`

Reply:
yeah, `^` would work if we had a 1.0 release but alas...